packages/fxa-settings/src/components/Settings/PageSecurityEvents/SecurityEvent.tsx

```diff
@@ -0,0 +1,41 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from 'react';
+import { FtlMsg } from 'fxa-react/lib/utils';
+
+export function SecurityEvent({
+  name,
+  createdAt,
+  verified,
+}: {
+  name: string;
+  createdAt: number;
+  verified?: boolean;
+}) {
+  const createdAtDateText = Intl.DateTimeFormat('en-US', {
+    year: 'numeric',
+    month: 'numeric',
+    day: 'numeric',
+    hour: 'numeric',
+    minute: 'numeric',
+  }).format(new Date(createdAt));
+
+  const formattedName = `security-events-${name.split('.').join('-')}`;
+
+  return (
+    <li className="mt-5 ml-4" data-testid={formattedName}>
+      <div className="absolute w-3 h-3 bg-green-600 rounded-full mt-1.5 -left-1.5 border border-green-700"></div>
```

Comment:
Instead of an extra DOM element here, how 'bout a pseudo? 

`<li className="mt-5 ml-4 before:content-[''] before:bg-green-600 before:w-3 before:h-3 before:rounded-full before:absolute before:-left-1.5 before:top-1.5">`

Reply:
Tried this but it wasn't rendering the subsequent items properly. Punting since we should probably update the design later.

Reply:
Ah oops, I think it just needs a `relative` class added on it (and `relative` on the parent `ol` could be removed) and a different `left-` value but sounds fine to me if we're planning on revisiting later anyways.